GameOfLifeApp/Controllers/GameManager.cs

```diff
@@ -66,5 +68,88 @@ public void RunGame(IGameField field, bool isNewGame)
+        public void RunMultipleGames(List<IGameField> games, IGameField field, bool isNewGame)
+        {
+            if (isNewGame)
+            {
+                int countOfRows = _communicator.GetBoundedResponse(Messages.userInputOfCountOfRows, minCountOfRowsOrColumns, maxCountOfRowsOrColumns);
+                int countOfColumns = _communicator.GetBoundedResponse(Messages.userInputOfCountOfColumns, minCountOfRowsOrColumns, maxCountOfRowsOrColumns);
+
+                for (int gameNumber = 0; gameNumber < 1000; gameNumber++)
+                {
+                    IGameField gameField = new GameField();
+                    gameField.CountOfRows = countOfRows;
+                    gameField.CountOfColumns = countOfColumns;
+                    field.CountIteration = 0;
+                    _gameLogic.SetUpField(gameField);
+                    games.Add(gameField);
+                }
+            }
+
+            List<int> gamesIdsToShow = _communicator.GetGamesId("Please choose 8 games for displaying.");
+            List<IGameField> gameFieldsToShow = new List<IGameField>();
+
+            for (int i = 0; i < gamesIdsToShow.Count; i++)
+            {
+                gameFieldsToShow.Add(games[gamesIdsToShow[i]]);
+            }
+
+            while (true)
+            {
+                Console.Clear();
+                Console.SetCursorPosition(0, 0);
+
+                int displayedAliveCellsCount = 0;
+                int totalAliveCellsCount = 0;
+                int totalLiveGamesCount = 0;
+
+                field.CountIteration++;
+
+                foreach (var gameField in games)
+                {
+                    _gameLogic.GetNextGeneration(gameField);
+                    _gameLogic.CountAliveCells(gameField);
+                    int aliveCellsCount = _gameLogic.CountAliveCells(gameField);
+                    totalAliveCellsCount += aliveCellsCount;
+
+                    if (aliveCellsCount > 0)
+                    {
+                        totalLiveGamesCount++;
+                    }
+                }
+
+                for (int i = 0; i < gamesIdsToShow.Count; i++)
+                {
+                    _gameLogic.CountAliveCells(gameFieldsToShow[i]);
+                    int aliveCellsCount = _gameLogic.CountAliveCells(gameFieldsToShow[i]);
+                    displayedAliveCellsCount += aliveCellsCount;
+                }
+
+                _display.ShowMultipleIteration(gameFieldsToShow);
+
+                Console.WriteLine($"The count of iterations is {field.CountIteration}.");
+                Console.WriteLine($"The count of alive cells in 8 displayed games is {displayedAliveCellsCount}.");
+                Console.WriteLine($"The count of alive cells in total is {totalAliveCellsCount}.");
+                Console.WriteLine($"The count of live games is {totalLiveGamesCount}.");
+                if (totalAliveCellsCount < 1)
```

Comment:
I'd add empty lines before and after if code block.

Reply:
Done.